labs/3.md

```diff
@@ -1,13 +1,23 @@
 <a id="markdown-argocd-deployment" name="argocd-introduction"></a>
-# Argo CD Introduction
+# Git Ops Introduction
+Argo CD is a GitOps tool for deploying and managing Kubernetes services. The term GitOps is applied to applications or tools that are used to the deploy YAML files that are stored within a git repository. This repository can be private repository on GitHub or GitLab, on premise, or within a public git repository.  GitOps tools can be triggered to deploy or update Kubernetes objects either manually, through a webhook, or automatically. Some GitOps tools even offer the ability to prune or remove resources automatically that are not currently defined wtihin the git repository.
```

Comment:
are not currently defined wtihin the git repository

s/wtihin/within/g

Reply:
tools that are used to the deploy YAML files

Do we want a more generic definition of GitOps or a K8s specific definition? I'd add something like:

The term GitOps on Kubernetes is applied to...